source/content/guides/object-cache-pro/02-installing-configuring.md

```diff
@@ -341,6 +343,23 @@ You must add a line to your `composer.json` file if you have Composer-managed Wo
       "web/app/themes/{$name}/": ["type:wordpress-theme"]
     },
 	```
+## Known Issues and Workarounds
+### `terminus install:run <site>.<env> ocp` fails to install Object Cache Pro on _subdomain multisites_
+While most of the time, the Terminus command runs on single or WordPress multisites without issues, we have observed the workflow failing when _the Pantheon platform url_ (e.g. `dev-mysite.pantheonsite.io`) does not match the _site URL stored in the database_. This might happen if vanity domains are configured for the lower environments (e.g. `dev.mysite.com`). This can be observed when the link provided to the site from the Pantheon dashboard does not match the WordPress site URL. To check the URL stored in the database, you can run `terminus wp -- <site>.<env> option get siteurl`.
+
+#### Workaround
+The workaround for getting the Terminus command to succeed when the Pantheon platform URL does not match the URL stored in the database is to temporarily replace the vanity URL in the database with the platform URL, running the Terminus command again to install Object Cache Pro, and then re-running the search & replace (or restoring a database backup).
+
+1. Make a [database backup](https://docs.pantheon.io/guides/backups/create-backups) before you begin.
+1. Run the `wp search-replace` command via Terminus:
+
+		```bash{promptUser: user}
+		terminus wp -- <site>.<env> search-replace <old-url> <new-url> --all-tables --network
+		```	
+	
+	In this example the `<old-url>` represents your custom **vanity domain** and the `<new-url>` you are changing it _to_ represents the Pantheon platform domain (e.g. `dev-mysite.pantheonsite.io`). You do not need to include the protocols (e.g. `http`/`https`) in the URLs. You may wish to append `--dry-run` to the command initially if you would like to test the command and see how many changes will be made before running.
+1. Re-run the `terminus install:run <site>.<env> ocp` command and wait for the workflow to finish.
+1. After the workflow has finished, assuming Object Cache Pro was installed successfully, you can re-run the `search-replace` command above, swapping the `<old-url>` and `<new-url>` values, or you may simply [restore your backup](https://docs.pantheon.io/guides/backups/restore-from-backup)
```

Comment:
1. We should be more explicit that this workflow/workaround is for non-live environments
1. rather than an "undo" search-replace, if they have WPMS search-replace configured, we might suggest copying content from live again allowing search-replace to take effect.

Reply:
The copying content from live only works if they _have_ a live environment. If they never created it, they won't be able to do that. But otherwise agree.

Reply:
Updated